program/scripts/PDBminer_functions.py

```diff
@@ -192,7 +192,37 @@ def get_structure_metadata(pdb_id):
         resolution = dictionary_exp['resolution']
     
     return deposition_date, experimental_method, resolution
+
+def get_PDBredo(pdb, method):
+    print(f"FUNCTION: get_PDBredo({pdb}, {method})")
+    """
+
+    Parameters
+    ----------
+    pdb : four letter PDB code.
+    method: the experimental method used to obtain PDB.
+
+    Returns
+    -------
+    str: YES/NO for availability in PDB-REDO database.
+    rfree_improve: a string detailing the orignal and PDBredo r-free values, "N/A" if non applicable
+    """
+    
+    if method in ['X-RAY DIFFRACTION','ELECTRON MICROSCOPY','ELECTRON CRYSTALLOGRAPHY']:
+        url = f"https://pdb-redo.eu/db/{pdb}/data.json"
+        response = requests.get(url)
+        
+        if response.status_code == 200:
+            response_data = response.json()
+            r_free_pdb = response_data['properties']['RFREE']
+            r_free_pdbredo = response_data['properties']['RFFIN']
+            rfree_improve = f"{r_free_pdb};{r_free_pdbredo}"
```

Comment:
are we really bent towards having one single column? Unless someone really reads the docs carefully, they will be prone to think that these are different R-free from PDBREDO or just make a guess. I think having it in two columns (with explicit naming on what they are) would be more clear. I don't think one extra column will change the size of the dataframe much at this point

Reply:
I Have removed the original r-free. A better source for this value is the mmcif dictionary because it is independent of PDB-REDO database. So this we be introduced in the next PR.